parcels/field.py

```diff
@@ -600,87 +628,6 @@ def cell_areas(self):
             self.calc_cell_edge_sizes()
         return self.grid.cell_edge_sizes['x'] * self.grid.cell_edge_sizes['y']
 
-    def search_indices_vertical_z(self, z):
```

Comment:
the remaining code deletions are - I assume - just done for reasons of simplification and overview, and not because it's not possible to do.

Reply:
Some of these functions actually have not been replaced, but moved to different classes. In this case `search_indices_vertical_z` is now part of the `BaseZGrid` implementation that can be found in `parcels.numba.grid.zgrid`.